pynecone/components/forms/advanced_select.py

```diff
@@ -0,0 +1,365 @@
+    # default: "color"
+    selected_option_style: Var[str]
+
+    # [chakra]
+    # The size of the component.
+    # options: "sm" | "md" | "lg"
+    # default: "md"
+    size: Var[str]
+
+    # Sets the tabIndex attribute on the input
+    tab_index: Var[int]
+
+    # Select the currently focused option when the user presses tab
+    tab_selects_value: Var[bool]
+
+    # [chakra]
+    # Variant of multi-select tags
+    # options: "subtle" | "solid" | "outline"
+    # default: "subtle"
+    tag_variant: Var[str]
+
+    # Remove all non-essential styles
+    unstyled: Var[bool]
+
+    # [chakra]
+    # If this prop is passed,
+    # the dropdown indicator at the right of the component will be styled
+    # in the same way the original Chakra Select component is styled,
+    # instead of being styled as an InputRightAddon.
+    # The original purpose of styling it as an addon
+    # was to create a visual separation between the dropdown indicator
+    # and the button for clearing the selected options.
+    # However, as this button only appears when isMulti is passed,
+    # using this style could make more sense for a single select.
+    # default: False
+    use_basic_style: Var[bool]
+
+    # [chakra]
+    # The variant of the Select. If no variant is passed,
+    # it will default to defaultProps.variant from the theme for Chakra's Input component.
+    # If your component theme for Input is not modified, it will be outline.
+    # options: "outline" | "filled" | "flushed" | "unstyled"
+    # default: "outline"
+    variant: Var[str]
+
+    @classmethod
+    def get_controlled_triggers(cls) -> Dict[str, Var]:
+        """Get the event triggers that pass the component's value to the handler.
+
+        Returns:
+            A dict mapping the event trigger to the var that is passed to the handler.
+        """
+        return {
+            "on_change": EVENT_ARG.target.value,
+        }
+
+    @classmethod
+    def get_alias(cls) -> str:
+        """Avoid conflict with default Chakra Select.
+
+        Returns:
+            The alias string
+        """
+        return "AdvancedSelect"
+
+    @classmethod
+    def create(
+        cls, options: List[Component | str | int | float | bool], **props
+    ) -> Component:
+        """Takes a list of options and additional properties, checks if each option is an
+        instance of AdvancedOption, and returns a Select component with the given options and
+        properties. No children allowed.
+
+        Args:
+            options (List[Component | str | int | float | bool]): A list of values that can be of type
+        Component(AdvancedOption), str, int, float or bool.
+
+        Returns:
+            The `create` method is returning an instance of the `AdvancedSelect` class.
+        """
+        checked_options = []
+        for option in options:
+            if not isinstance(option, AdvancedOption):
+                checked_options.append(AdvancedOption.create(label=str(option)))
```

Comment:
Looks like the `options` prop expects a list of dicts which his the main problem